docs/src/languages/php.md

```diff
@@ -2,34 +2,54 @@
 
 PHP support is available through the [PHP extension](https://github.com/zed-extensions/php).
 
-- Tree-sitter: https://github.com/tree-sitter/tree-sitter-php
-- Language Servers:
-  - [phpactor](https://github.com/phpactor/phpactor)
-  - [intelephense](https://github.com/bmewburn/vscode-intelephense/)
+- Tree-sitter: [tree-sitter/tree-sitter-php](https://github.com/tree-sitter/tree-sitter-php)
+- Language Server: [phpactor/phpactor](https://github.com/phpactor/phpactor)
+- Alternate Language Server: [bmewburn/vscode-intelephense](https://github.com/bmewburn/vscode-intelephense/)
 
-## Choosing a language server
+## Install PHP
 
-The PHP extension offers both `phpactor` and `intelephense` language server support.
+The PHP extension requires PHP to be installed and available in your path:
 
-`phpactor` is enabled by default.
+```sh
+# macOS via Homebrew
+brew install php
 
-### Phpactor
+# Debian/Ubuntu
+sudo apt-get install php-cli
 
-The Zed PHP Extension can install `phpactor` automatically but requires `php` to be installed and available in your path:
+# CentOS 8+/RHEL
+sudo dnf install php-cli
 
-```sh
-# brew install php            # macOS
-# sudo apt-get install php    # Debian/Ubuntu
-# yum install php             # CentOS/RHEL
-# pacman -S php               # Arch Linux
+# Arch Linux
+sudo pacman -S php
+
+# check PHP path
 which php
 ```
 
+## Choosing a language server
+
+The PHP extension offers Phpactor and Intelephense as language server with Phpactor as default.
+
+### Phpactor
+
+To use Phpactor instead of Intelephense, add the following to your `settings.json`:
+
+```json [settings]
+{
+  "languages": {
+    "PHP": {
+      "language_servers": ["phpactor", "!intelephense", "..."]
+    }
+  }
+}
+```
+
 ### Intelephense
 
-[Intelephense](https://intelephense.com/) is a [proprietary](https://github.com/bmewburn/vscode-intelephense/blob/master/LICENSE.txt#L29) language server for PHP operating under a freemium model. Certain features require purchase of a [premium license](https://intelephense.com/).
+[Intelephense](https://intelephense.com/) is a [proprietary](https://github.com/bmewburn/vscode-intelephense/blob/master/LICENSE.txt#L29) language server for PHP operating under a freemium model. Certain features require purchase of a [premium license](https://intelephense.com/buy).
 
-To switch to `intelephense`, add the following to your `settings.json`:
+To use Intelephense, add the following to your `settings.json`:
```

Comment:
This would be a good place to use the wording from https://github.com/zed-industries/zed/pull/44001/files#r2584841567